toolkit/pythonit_toolkit/pastaporto/tokens.py

```diff
@@ -24,3 +24,20 @@ def decode_service_to_service_token(
         algorithms=["HS256"],
         options={"require": ["exp", "iss", "aud", "iat"]},
     )
+
+
+def generate_service_to_service_token(
+    secret, issuer: str, audience: str, expires_in: str = "1m"
```

Comment:
let's not accept the expiration as param, we don't really want long duration tokens